content/sensu-go/6.7/plugins/featured-integrations/ansible.md

```diff
@@ -32,7 +32,8 @@ The [Sensu Ansible Handler plugin][4] supports both Ansible Tower and Ansible AW
 
 ## Get the plugin
 
-For a turnkey experience with the Sensu Ansible Handler plugin, use our curated, configurable [quick-start template][3] to integrate Sensu with your existing Ansible Tower workflows.
+For a turnkey experience with the Sensu Ansible Handler plugin, use the Sensu web UI [Catalog][10] to configure and install it.
+Or, use our curated, configurable [quick-start template][3] to integrate Sensu with your existing Ansible Tower workflows.
```

Comment:
Part 2 of directing people to the Catalog for integrations that are available there...